packages/client/src/containers/ContactUs.container.js

```diff
@@ -34,10 +36,15 @@ export const ContactUs = () => {
     if (Object.keys(formErrors).length === 0 && isSubmit) {
       console.log(formValues);
     }
```

Comment:
Please remove this effect and remove all the console.log from this file and the eslint disables.